### ISSUE TITLE
Some accessibility issues

### ISSUE THREAD

**Issue by mgifford** [colbyfayock/imagecarbon#4]
Neat site btw. Like how you've leveraged CO.js to do this. 

I wanted to note that there are some accessibility issues on the site. Most are like this.

Thanks! 

Mike

Title: WCAG 2.4.4,WCAG 4.1.2: Ensures links have discernible text (a)
Tags: Accessibility, WCAG 2.4.4, WCAG 4.1.2, link-name

Issue: Ensures links have discernible text (link-name - https://accessibilityinsights.io/info-examples/web/link-name)

Target application: Results for civicactions.com - Image Carbon - https://www.imagecarbon.com/sites/civicactions.com

Element path: .Footer_cloudinaryLogo__DXBo5 > a

Snippet: <a href="https://cloudinary.com/?utm_source=imagecarbon.com&amp;utm_medium=referral&amp;utm_campaign=devx_imagecarbon&amp;utm_content=footer_cloudinary_logo">

How to fix: 
Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

Environment: Microsoft Edge version 112.0.1722.39

====

This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by colbyfayock** [comment]
thanks again @mgifford will work to get these resolved

**Comment by colbyfayock** [comment]
downloaded that extension and resolved the fastpass issues! cool extension
